browser/browser_context_keyed_service_factories.cc

```diff
@@ -53,6 +53,8 @@
 #endif
 
 #if BUILDFLAG(ENABLE_IPFS)
+#include "brave/browser/brave_wallet/brave_wallet_auto_pin_service_factory.h"
```

Comment:
After https://github.com/brave/brave-core/commit/d51780705214bd5c5a2c2806a49dfeecd7210a60#diff-87dbeae647f32aa0670c977a84d99be9862a63bf9cc1f0a8b191da86f1d1a21f
we should consider removing ENABLE_IPFS build flag. But not in this PR I think